src/IO/AzureBlobStorage/copyAzureBlobStorageFile.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "config.h"
+
+#if USE_AZURE_BLOB_STORAGE
+
+#include <Storages/StorageAzureBlobCluster.h>
+#include <Storages/StorageAzureBlob.h>
+#include <Interpreters/threadPoolCallbackRunner.h>
+#include <base/types.h>
+#include <functional>
+#include <memory>
+
+
+namespace DB
+{
+class SeekableReadBuffer;
+
+using CreateReadBuffer = std::function<std::unique_ptr<SeekableReadBuffer>()>;
+
+/// Copies a file from AzureBlobStorage to AzureBlobStorage.
+/// The parameters `src_offset` and `src_size` specify a part in the source to copy.
+void copyAzureBlobStorageFile(
+    std::shared_ptr<Azure::Storage::Blobs::BlobContainerClient> src_client,
+    std::shared_ptr<Azure::Storage::Blobs::BlobContainerClient> dest_client,
+    const String & src_bucket,
+    const String & src_key,
```

Comment:
What are the meaning of `bucket` and `key` in case of AzureBlobStorage?

Reply:
Renamed Bucket-Key to Container-Blob